src/post.ts

```diff
@@ -155,6 +159,28 @@ function stillsuit() {
   }
 }
 
+function funguySpores() {
+  if (
+    myLevel() >= 15 &&
```

Comment:
possibly add a comment about why this; we should fiddle with this threshold a bit